proposals/0448-type-variable-scoping.rst

```diff
@@ -429,8 +527,6 @@ Proposed Change Specification
    it does not trigger any behavior associated with partial type signatures.
    In particular, ``-XPartialTypeSignatures`` is not necessary, and no diagnostic is produced.
 
-#. As with term variables, it is an error to bring the same type variable into scope in two (or more) places within the same pattern.
-
```

Comment:
Why did you remove this?

Reply:
Oh whoops, I accidentally pushed my merging of https://github.com/ghc-proposals/ghc-proposals/pull/556 into this to fix comments.

That one deleted this because it is not quite correct and the expanded above portion now covers it.